shared/src/commonMain/kotlin/org/hyperskill/app/study_plan/domain/analytics/stage_implement_unsupported_modal/StageImplementUnsupportedModalClickedGoToHomeScreenHyperskillAnalyticEvent.kt

```diff
@@ -12,18 +12,16 @@ import org.hyperskill.app.analytic.domain.model.hyperskill.HyperskillAnalyticTar
  * JSON payload:
  * ```
  * {
- *     "route": "/projects/{PROJECT_ID}/stages/{STAGE_ID}/implement",
+ *     "route": "/study-plan",
  *     "action": "click",
  *     "part": "stage_implement_unsupported_modal",
  *     "target": "go_to_home_screen"
  * }
  * ```
  * @see HyperskillAnalyticEvent
  */
-class StageImplementUnsupportedModalClickedGoToHomeScreenHyperskillAnalyticEvent(
-    route: HyperskillAnalyticRoute
-) : HyperskillAnalyticEvent(
-    route,
+object StageImplementUnsupportedModalClickedGoToHomeScreenHyperskillAnalyticEvent : HyperskillAnalyticEvent(
```

Comment:
Usually feature analytic event starts (has a prefix of a feature name) `StageImplementUnsupportedModalClickedGoToHomeScreenHyperskillAnalyticEvent` -> `StudyPlanStageImplementUnsupportedModalClickedGoToHomeScreenHyperskillAnalyticEvent `

Reply:
Very long names 😄 I think in the future we could change our style guide for naming...

As far as I remember we have such long names because of iOS analytics for notifications

Reply:
I returned it to classes, it's better to use the same approach for all analytic events